README.md

```diff
@@ -9,7 +9,7 @@ We are going to continue with our search engine that pulls identifying informati
 
 ### Your task
 
-In the last section, we pulled the header, the text from the first link and the `href` from the first link. In this section we are going to pull the same information, but from a different simplified Wikipedia page.
+In the last section, we pulled the header, the text from the first link and the `href` from the first link. In this section we are going to pull the same information, but from a different simplified Wikipedia page. Since this is a **lab** we are going to submit our final code in the CodePen below. You will work in the Console to discover what the final solution will be, but to complete the lab, you must put your completed code in the CodePen until in the Console you see the correct answers.
```

Comment:
The output from the console should be the desired output. Use the console to discover what the correct answers to be. Once you have your answer, you must mark it down in the JS tab to solve the lab.